adminSiteServer/app.test.ts

```diff
@@ -193,11 +193,10 @@ async function makeRequestAgainstAdminApi(
         body,
     })
 
-    expect(response.status).toBe(200)
-
     const json = await response.json()
 
     if (verifySuccess) {
+        expect(response.status).toBe(200)
```

Comment:
this seems like a reasonable change to make, but it kind of begs the question: why have `success = true` and `status = 200`

🤔